app-samples/Face/FaceApiSnapshotSample/FaceApiSnapshotSample/Program.cs

```diff
@@ -0,0 +1,266 @@
+﻿namespace FaceApiSnapshotSample
```

Comment:
This one seems to be compatible with auto-test framework, any reason to be a standalone application sample?

Reply:
For snapshot feature, there is another required user provided id `Azure Cognitive Service Face subscription id`. The existing samples for auto-test needs only `Azure Cognitive Service Face subscription key`.

Reply:
okay, I believe this should not be a limitation. But I need some tests.
I'll let the code in first.

Reply:
Thanks. As I said, one more parameter should be passed into the snapshot sample if you adapt it to existing auto-test framework.